evm-contracts/test/v0.6/PrepaidAggregator.test.ts

```diff
@@ -1679,4 +1681,149 @@ describe('PrepaidAggregator', () => {
       matchers.bigNum(originalBalance.add(deposit), newBalance)
     })
   })
+
+  describe('#startNewRound', () => {
+    beforeEach(async () => {
+      await aggregator
+        .connect(personas.Carol)
```

Comment:
Just to clarify: This means the transaction is signed by Carol, right?

Reply:
Yes.